### PR TITLE
Readme fixes

### PR DESCRIPTION
**Pull request by Wumpf**
### What

* Fix accidental h1 heading on CONTRIBUTING.md
* Various small updates to README.md

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6308?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/6308?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!

- [PR Build Summary](https://build.rerun.io/pr/6308)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,9 +14,9 @@
 
 # Build time aware visualizations of multimodal data
 
-Use the Rerun SDK (available for C++, Python and Rust) to log data like images, tensors, point clouds, and text. Logs are streamed to the Rerun Viewer for live visualization or to file for later use.
+Use the Rerun SDK (available for C++, Python and Rust) to log data like images, tensors, point clouds, and text. Logs are streamed to the Rerun viewer for live visualization or to file for later use.
 
-* [Run the Rerun Viewer in your browser](https://www.rerun.io/viewer)
+* [Run the Rerun viewer in your browser](https://www.rerun.io/viewer)
```
Comment (emilk) [inline README.md:19]: We capitalize product names, and "Rerun Viewer" counts as a product (currently). See `DESIGN.md`

Comment (emilk) [reply README.md:19]: …though I see we do so very inconsistently and don't enforce it in lint.py

Comment (Wumpf) [reply README.md:19]: handled in https://github.com/rerun-io/rerun/pull/6317

**Review by abey79 — APPROVED** [review]

**Comment by Wumpf** [comment]
(merged in #6317 and resolved conflicts, now waiting for green ci)

**Pull request merged**